templates.html

```diff
@@ -0,0 +1,203 @@
+<div class="qa-templates" style="display: none">
+
+    <div class="js-page-hdr">
+        <div class="qa-tbl-row qa-tbl-hdr">
+            <div class="qa-page-id qa-center">
+                <span>Page ID</span>
+            </div>
+            <div class="qa-page-title">
+                <span>Page Title</span>
+            </div>
+            <div class="qa-page-q-count qa-center">
+                <span>Question Count</span>
+            </div>
+            <div class="qa-page-preview qa-center">
+                <span>Preview</span>
+            </div>
+        </div>
+    </div>
+
+    <div class="js-page-row">
+        <div class="qa-tbl-row page js-qa-page-row">
+            <div class="qa-page-id qa-center">
+                <span></span>
+            </div>
+            <div class="qa-page-title">
+                <span></span>
+            </div>
+            <div class="qa-page-q-count qa-center">
+                <span></span>
+            </div>
+            <div class="qa-page-preview qa-center">
+                <button type="button" class="btn btn-success page-preview">
+                    <span class="glyphicon glyphicon-eye-open"></span>
+                    <a href="#link_to_page" target="_blank" rel="noopener"></a>
```

Comment:
Dashes should be preferred to underscores inside links. So:

```html
<a href="#preview-of-page"></a>
```

is better than:

```html
<a href="#preview-of-page"></a>
```

Reference: https://www.mattcutts.com/blog/dashes-vs-underscores/